launch.py

```
@@ -203,9 +416,19 @@ def create_venv(venv_path: Path, recreate: bool = False):
         logger.info(f"Virtual environment already exists at {venv_path}")
 
 
+<<<<<<< HEAD
```

Pylint (4.0.0)

[error] Parsing failed: 'invalid syntax (launch, line 419)'

(E0001)